marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/graphLeafNodes.cy.tsx

```diff
@@ -105,11 +105,8 @@ describe("Leaf Nodes", () => {
       graphExplore.stopStabilization();
       graphView.physicsAnimationToggle();
     });
-  });
-  // TODO: COMMENTED SINCE WE NEED TO PERFORM A CLICK AND HOVER TO MAKE THE TOOLTIP APPEARS
-  // AND THAT COLLAPSES THE PRODUCT NODE, AND THE LEAF NODE DISAPPEARS
-  // Wait needed for the graph to get stabilized
-  /*cy.wait(2000);
+
+    // Wait needed for the graph to get stabilized
```

Comment:
Can we remove this comment? I don't see any wait here

Reply:
Removed!